proto/call.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2014 The Cockroach Authors.
```

Comment:
can you do the move in one commit and the changes in another? that'll make review easier.